### ISSUE TITLE
A11y of map switch on desktop

### ISSUE THREAD

**Issue by phillimorland** [liqd/a4-meinberlin#2703]
**URL:** /project
**user:** SR users
**expected behaviour:** map switch is tababble 
**behaviour:** not tababble and cannot switch when using keyboard navigation
**important screensize:** 1020 up
**device & browser:** all
**Comment/Question:** attempt to refactor and fix in following pull request but too much of time sink, will come back to it over xmas

Screenshot?
